src/spicy/spicyz/glue-compiler.cc

```diff
@@ -343,7 +343,7 @@ hilti::Result<std::string> GlueCompiler::getNextEvtBlock(std::istream& in, int*
     std::string chunk;
 
     // Parser need to track whether we are inside a string or a comment.
-    enum State { Default, InComment, InString } state = Default;
+    enum State : char { Default, InComment, InString } state = Default;
```

Comment:
Total drive-by, I agree this can be useful if there's lot of enum instances, but here there's exactly one, so this seems just extra noise, but I'm happy to learn if/whether we should do this more often.

Reply:
I came at this from the angle: my editor yells at me whenever I open this file since this lint is enabled in the default clang-tidy config. I personally do not think this is more noisy than a local suppression.